source/Halibut.Tests/Halibut.Tests.csproj

```diff
@@ -2,13 +2,17 @@
 
   <PropertyGroup>
     <VersionPrefix>0.0.0</VersionPrefix>
-    <TargetFrameworks>net462;netcoreapp2.2</TargetFrameworks>
     <AssemblyName>Halibut.Tests</AssemblyName>
     <PackageId>Halibut.Tests</PackageId>
     <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
     <GenerateAssemblyTitleAttribute>false</GenerateAssemblyTitleAttribute>
   </PropertyGroup>
-
+  <PropertyGroup Condition="!$([MSBuild]::IsOSUnixLike())">
+	  <TargetFrameworks>netcoreapp2.2;net472</TargetFrameworks>
```

Comment:
Did we need to upgrade this to net472? I'm actually surprised it was on net462 to start with. I think its better if the test project has the same targets as the project under test, so that you are actually testing the same thing :)

Reply:
Good point - I'll revert it to net45 to line it up with the assembly under test.